packages/config/src/lib/createPlugins.ts

```diff
@@ -11,12 +12,19 @@ const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin'
 const { glob } = require('glob');
 const path = require('path');
 
+const chromeIndexHtmlUrl = (env = 'dev-stable') => {
```

Comment:
Can't use this. We will be removing the build repositories. Old build pipeline is coming to an end. We will be suing the container images as the source of truth for local chrome.